R/makeBiObjBBOBFunction.R

```diff
@@ -25,6 +26,8 @@
 #' for a detailed description of the bi-objective BBOB functions.
 #' An overview of which pair of single-objective BBOB functions creates
 #' which of the 55 bi-objective BBOB functions can be found \href{http://numbbo.github.io/coco-doc/bbob-biobj/functions/#the-bbob-biobj-test-functions-and-their-properties}{here}.
+#' A full description of the extended suite with 92 functions can be found
+#' \href{http://numbbo.github.io/coco-doc/bbob-biobj/functions/#the-extended-bbob-biobj-ext-test-suite-and-its-functions}{here}.
```

Comment:
Please add "speaking links", i.e., ... can be found in the official documentation.